app/pastes/update-paste/updatePasteValidator.js

```diff
@@ -0,0 +1,51 @@
+const visibilityAcceptedValues = require('../../entities/paste-entity/visibility');
+
+class UpdatePasteValidator {
+    validate({id, name, text, visibility}) {
+        const errors = [];
+
+        if (!id && !name && !text && !visibility) {
+            errors.push('Parameters are required.');
+        }
```

Comment:
Same here. Consider using `||` or split it into separate checks.
It is a good practice to 
`if (!id) {
    ...
 } else {
      if (id.length ....) {}
      if (/xcxcx/.test(id)) {}
      ....
 }`